en/docs/testing/test-graphql-endpoints-via-the-graphql-console.md

```diff
@@ -32,5 +32,8 @@ Follow the steps below to test a GraphQL endpoint via the GraphQL Console:
 
     !!! tip
         If you want to generate a required query or mutation for testing via the GraphQL Explorer, click **Explorer** to open the GraphQL Explorer.
+    
+    !!! note
+        GraphQL Explorer has a known issue where the list inputs are not constructed properly. You can update the generated query before executing when list inputs are present.
```

Comment:
Suggestion:
GraphQL Explorer has a known issue causing improper construction of list inputs. Therefore, if a query has list inputs, you must manually update the generated query before executing it.

Reply:
Shall we also mention that the user should only update the list input part due to this limitation?